### DISCUSSION TITLE
Cannot get the original image back after editing an image

### DISCUSSION THREAD

**Discussion by dschinkel** [community/community#113112] [Copilot Conversations]
### Select Topic Area

Question

### Body

There's no way to get my original image that DALLE gened back!  I make a modification to it, and now I just see the modification. It's like it lost layers!

Original
<img width="673" alt="2682" src="https://github.com/community/community/assets/338798/cc56ed1d-c6af-4064-af7f-342fb3d2460b">

After I modified it
<img width="501" alt="Screenshot 2024-03-17 at 9 41 32 PM" src="https://github.com/community/community/assets/338798/a579d532-b401-41ea-afa6-9235eeb287f6">

I don't see a way to get back to the original.  It will only go back as far as the last layer modification which is they grey with just the top of the tomato having color.

I also do not see a way to say "generate an image like this one that I previously generated with this new prompt"

**Comment by davecheney** [comment]
I think you are referring to the OpenAI product, not Copilot. Closing.

**Comment by dschinkel** [comment]
omg "CoPilot" not "CodePilot" 🤣🤣
